lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -377,6 +377,17 @@ module Resources
           },
           "status": "accepted"
         }
+      ],
+      "reasonCode": [
+        {
+          "coding": [
```

Comment:
leave coding out of examples as we consume text only

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/063202ecf7b52c89b017b9b8d20412808d417975#diff-1a83ec9f4ba450cba6a16d8bcda1890cL383)

Reply:
take it out of all the examples including in appointment.yaml

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/74138237b033684b95e9ebd89bbd163b23aa2d59#diff-fbb215a73c336ca1ac34d91de2fb411eL76)

Reply:
updated again [here](https://github.com/cerner/fhir.cerner.com/commit/74138237b033684b95e9ebd89bbd163b23aa2d59#diff-fbb215a73c336ca1ac34d91de2fb411eL81)